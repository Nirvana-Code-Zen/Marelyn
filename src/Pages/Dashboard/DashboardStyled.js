import styled from 'styled-components'

const Dashboard = styled.div`
  display: grid;
  padding: 1rem;
  grid-template-columns: repeat(3, 1fr);
  grid-template-rows: repeat(2,1fr);
  grid-gap: 1rem;
`

export const ProductsSection = styled.div`
  display: flex;
  justify-content: center;
  grid-column: 1/2;
  grid-row: 1/2;

`
export const SalesSection = styled.div`
  display: flex;
  justify-content: center;
  grid-column: 3/4;
  grid-row: 1/3;
`
export const ReportsSection = styled.div`
  display: flex;
  justify-content: center;
  grid-column: 1/3;
  grid-row: 2/3;
`

export const OffertSection = styled.div`
  display: flex;
  justify-content: center;
  grid-column: 2/3;
  grid-row: 1/2;
`
export default Dashboard
