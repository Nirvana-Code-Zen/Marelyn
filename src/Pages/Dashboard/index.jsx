import DashboardStyled, { ProductsSection, SalesSection, ReportsSection, OffertSection } from './DashboardStyled'
import { H4 } from '../../Global-styles/Components/Titles'

const Dashboard = () => {
  return (
    <DashboardStyled Styled className='boxshadow'>
      <ProductsSection className='boxshadow'>
        <H4>Seccion de productos nuevos</H4>
      </ProductsSection>
      <SalesSection className='boxshadow'>
        <H4>Seccion de ventas </H4>
      </SalesSection>
      <ReportsSection className='boxshadow'>
        <H4>Seccion de reportes </H4>
      </ReportsSection>
      <OffertSection className='boxshadow'>
        <H4>Seccion de ofertas </H4>
      </OffertSection>
    </DashboardStyled>

  )
}

export default Dashboard
