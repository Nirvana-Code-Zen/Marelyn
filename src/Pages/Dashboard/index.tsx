import { H4 } from '../../Global-styles/Components/Titles'

import { DashboardStyled, ProductsSection, SalesSection, ReportsSection, OffertSection } from './DashboardStyled'

const Dashboard = () => {
  return (
    <DashboardStyled className='boxshadow'>
      <ProductsSection className='boxshadow'>
        <H4>Seccion de productos nuevos</H4>
      </ProductsSection>
      <SalesSection className='boxshadow'>
        <H4>Seccion de ventas </H4>
      </SalesSection>
      <ReportsSection className='boxshadow'>
        <H4>Seccion de reportes </H4>
      </ReportsSection>
      <OffertSection className='boxshadow'>
        <H4>Seccion de ofertas </H4>
      </OffertSection>
    </DashboardStyled>

  )
}

export default Dashboard
