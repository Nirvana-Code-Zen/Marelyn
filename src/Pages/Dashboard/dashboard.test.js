import Dashboard from './DashboardStyled'
import renderer from 'react-test-renderer'

describe('<Dashboard/>', () => {
  test('Should render the styles of the dashboard component', () => {
    const dashboard = renderer.create(<Dashboard/>).toJSON()
    expect(dashboard).toMatchSnapshot()
  })
})
