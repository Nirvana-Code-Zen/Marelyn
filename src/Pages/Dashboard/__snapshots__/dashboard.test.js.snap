// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard/> Should render the styles of the dashboard component 1`] = `
.c0 {
  display: grid;
  padding: 1rem;
  grid-template-columns: repeat(3,1fr);
  grid-template-rows: repeat(2,1fr);
  grid-gap: 1rem;
}

<div
  className="c0"
/>
`;
